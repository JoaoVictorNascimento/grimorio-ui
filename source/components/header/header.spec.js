import '../../../internals/test/helper';

import Header from './header-component';

/** @test {Header} */
describe('Header component', () => {
/** @test {Header#render} */
  describe('#render', () => {
    it('render correctly', () => {
      const wrapper = shallow(
        <Header user="teste@teste.com" onLogout={() => {}} />
      );
      expect(wrapper.length).toEqual(1);
    });
  });
});
