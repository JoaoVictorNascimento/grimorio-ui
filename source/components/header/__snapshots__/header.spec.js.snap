// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component render desktop version with items 1`] = `"<header class=\\"header\\" items=\\"[object Object],[object Object]\\"><button type=\\"button\\" class=\\"headerUser clean inverted small\\"><div class=\\"selectWrap\\" value=\\"false\\"><span class=\\"button\\"><i style=\\"vertical-align:middle;font-size:18px\\" class=\\"material-icons icon iconLeft\\" align=\\"middle\\">person</i><i style=\\"vertical-align:middle;font-size:20px\\" class=\\"material-icons icon iconRight\\" align=\\"middle\\">arrow_drop_down</i></span><ul style=\\"height:auto\\" class=\\"menu isUnder\\"><li class=\\"option\\"><i style=\\"vertical-align:middle;font-size:18px\\" class=\\"material-icons icon optionIcon\\" align=\\"middle\\">home</i><span class=\\"optionContent\\">Opção 3</span></li><li class=\\"option\\"><i style=\\"vertical-align:middle;font-size:18px\\" class=\\"material-icons icon optionIcon\\" align=\\"middle\\">person</i><span class=\\"optionContent\\">Opção 4</span></li></ul></div></button><button type=\\"button\\" class=\\"headerLogout clean inverted small\\">Logout<i style=\\"vertical-align:middle;font-size:18px\\" class=\\"material-icons icon iconRight\\" align=\\"middle\\">exit_to_app</i></button></header>"`;

exports[`Header component render desktop version without items 1`] = `"<header class=\\"header\\"><button type=\\"button\\" class=\\"headerUser clean inverted small\\"><span> <i style=\\"vertical-align:middle;font-size:18px\\" class=\\"material-icons icon iconLeft\\" align=\\"middle\\">person</i>  </span></button><button type=\\"button\\" class=\\"headerLogout clean inverted small\\">Logout<i style=\\"vertical-align:middle;font-size:18px\\" class=\\"material-icons icon iconRight\\" align=\\"middle\\">exit_to_app</i></button></header>"`;

exports[`Header component render mobile version 1`] = `"<header class=\\"header isMobile\\"><div class=\\"logo\\"></div><button type=\\"button\\" class=\\"headerLogout isMobile clean inverted small\\"><i style=\\"vertical-align:middle;font-size:18px\\" class=\\"material-icons icon iconRight\\" align=\\"middle\\">exit_to_app</i></button></header>"`;

exports[`Header component render with children 1`] = `"<header class=\\"header\\"><div>teste com filho</div></header>"`;
